config/default/assisted-service-patch-pvc-bucket-fileystem.yaml

```diff
@@ -0,0 +1,13 @@
+kind: PersistentVolumeClaim
```

Comment:
I don't think PVCs can be included in the bundle. 

```
Error: accumulating resources: 2 errors occurred:
	* accumulateFile error: "accumulating resources from '../default': '/go/src/github.com/openshift/origin/config/default' must resolve to a file"
	* accumulateDirector error: "recursed accumulation of path '/go/src/github.com/openshift/origin/config/default': no matches for OriginalId ~G_v1_PersistentVolumeClaim|assisted-installer|bucket-pv-claim; no matches for CurrentId ~G_v1_PersistentVolumeClaim|assisted-installer|bucket-pv-claim; failed to find unique target for patch ~G_v1_PersistentVolumeClaim|bucket-pv-claim"
```

That's why they are documented in the CSV: https://github.com/openshift/assisted-service/blob/master/config/manifests/bases/assisted-service-operator.clusterserviceversion.yaml#L44